Scriptblock/sbBrowser.ps1

```diff
@@ -0,0 +1,51 @@
+﻿
+# User browser artefact collection
+Write-Host -ForegroundColor Yellow "`tCollecting User Browser Artefacts"
+                       
+$Users = Get-ChildItem "$env:systemdrive\Users\" -Force | where-object {$_.PSIsContainer} | Where-object { $_.Name -ne "Public" -And $_.Name -ne "All Users" -And $_.Name -ne "Default" -And $_.Name -ne "Default User"} | select-object -ExpandProperty name
+
+# From https://github.com/mark-hallman/plaso_filters/blob/master/filter_windows.txt
+$ChromeFiles = @("Bookmarks", "Cookies", "Current Session", "Current Tabs", "Extension Rules", "Extension State", "Favicons", "History", "Last Session", "Last Tabs", "Preferences", "Shortcuts", "Top Sites", "Visited Links", "Web Data")
+
+Foreach ($User in $Users) {
+    ### Chrome ###
+    # Determine if user has Chrome installed
+    if ( Test-Path "$env:systemdrive\Users\$User\AppData\Local\Google\Chrome\User Data" ) {
+        # Find all Chrome profile directories
+        if ( (Test-Path variable:Triage) -and ($Triage -eq $True) ) {
+            $ChromeProfiles = @("Default")
+        } else {
+            $ChromeProfiles = Get-ChildItem "$env:systemdrive\Users\$User\AppData\Local\Google\Chrome\User Data" -recurse -include "Google Profile.ico" -force | Select-Object -Property Directory | ForEach-Object { Split-Path $_.Directory -Leaf }
+        }
```

Comment:
I was playing around with ways to implement collections - since there is a wide variety of browser artefacts that can be acquired (and getting them all can take some time). I'm not convinced this is the best way (below Test-Path variable:varname tests if the variable exists, which is cool!)